src/components/Navbar/NavItems.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React from 'react';
+import { Divider } from '@material-ui/core';
+import { User, FileText, Folder, Type, MessageCircle } from 'react-feather';
+import { NavItem, NavItemProps } from '../NavItem';
+import { CFC } from '../../typings/components';
+
+type NavItemsProps = {
+  onClick?: NavItemProps['onClick'];
+};
+
+const NavItems: CFC<NavItemsProps> = ({ onClick }) => {
+  return (
+    <>
+      <NavItem to="/about" onClick={onClick} icon={User}>
```

Comment:
About page is the `index.ts` in pages so should be `to="/"`

Reply:
My bad, I didn't check 😅